scrapi/harvesters/addis_ababa.py

```diff
@@ -6,13 +6,38 @@
 from __future__ import unicode_literals
 
 from scrapi.base import OAIHarvester
+from scrapi.base import helpers
 
+def format_uris_addis_ababa(*args):
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
+
+    for arg in args:
+        if arg and 'http://hdl.handle.net/123456789/' in arg[0]:
+            DOC_ID = arg[0].replace('http://hdl.handle.net/123456789/', '')
```

Comment:
usually in python variables in ALL_CAPS are constants, so this would be better as lower case (super small detail)